src/map/lua/lua_baseentity.cpp

```diff
@@ -9894,6 +9894,38 @@ void CLuaBaseEntity::recalculateAbilitiesTable()
     PChar->pushPacket(new CCharAbilitiesPacket(PChar));
 }
 
+/************************************************************************
+ *  Function: getPlayersInRange()
+ *  Purpose : Returns a Lua table of players within range of the base entity
+ *  Example : local players = npc:getPlayersInRange(50)
+ *  Notes   : if the passed argument is nil or 0, just returns all players in the zone
+ ************************************************************************/
+
+sol::table CLuaBaseEntity::getPlayersInRange(uint32 dist = 0)
+{
+    auto players = lua.create_table();
+
+    uint32 size = 0;
+    // clang-format off
+    zoneutils::GetZone(m_PBaseEntity->getZone())->ForEachChar([&](CCharEntity* PChar) {
```

Comment:
Format this as allman braces please

Reply:
They're all surrounded with the clang-format exception because clang-format straight up can't handle making lambda functions not look like ass 😢